src/Symfony/Validator/Metadata/Property/Restriction/PropertySchemaCollectionRestriction.php

```diff
@@ -81,7 +81,7 @@ public function supports(Constraint $constraint, ApiProperty $propertyMetadata):
     private function mergeConstraintRestrictions(Constraint $constraint, ApiProperty $propertyMetadata): array
     {
         $propertyRestrictions = [];
-        $nestedConstraints = method_exists($constraint, 'getNestedContraints') ? $constraint->getNestedContraints() : $constraint->constraints;
+        $nestedConstraints = method_exists($constraint, 'getNestedContraints') ? $constraint->getNestedContraints() : $constraint->getNestedConstraints();
```

Comment:
Isn't it the same now?

Reply:
the method exists is there for backward compatibility